src/pages-and-resources/discussions/app-config-form/apps/shared/messages.js

```diff
@@ -147,6 +147,16 @@ const messages = defineMessages({
     defaultMessage: 'Delete this topic?',
     description: 'Label for discussion topic delete popup allowing a user to delete a topic.',
   },
+  renameDefaultTopic: {
+    id: 'authoring.discussions.builtIn.renameDefaultTopic.label',
+    defaultMessage: 'Rename general topic',
```

Comment:
I do not think so 'general' is a variable. It will be confusion between general or additional topic without this label.   
<img width="969" alt="Screenshot 2021-06-01 at 7 56 45 PM" src="https://user-images.githubusercontent.com/79941147/120344963-80585680-c313-11eb-8caa-083d15b338ac.png">

Reply:
if we rename the general topic to "something else". It might be a little confusing if the heading still says "general"? Can confirm this with jonathan

Reply:
Sure. I'll ping jonathon

Reply:
I confirmed from jonathan. It is hard coded string.